src/test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol

```diff
@@ -80,10 +82,25 @@ contract BLSRegistryCoordinatorWithIndicesUnit is MockAVSDeployer {
         address newChurnApprover = address(uint160(uint256(keccak256("newChurnApprover"))));
         cheats.prank(serviceManagerOwner);
         cheats.expectEmit(true, true, true, true, address(registryCoordinator));
-        emit ChurnApproverUpdated(newChurnApprover);
+        emit ChurnApproverUpdated(churnApprover, newChurnApprover);
         registryCoordinator.setChurnApprover(newChurnApprover);
     }
 
+    function testSetEjector_NotServiceManagerOwner_Reverts() public {
+        address newEjector = address(uint160(uint256(keccak256("newEjector"))));
+        cheats.expectRevert("BLSRegistryCoordinatorWithIndices.onlyServiceManagerOwner: caller is not the service manager owner");
+        cheats.prank(defaultOperator);
+        registryCoordinator.setEjector(newEjector);
+    }
+
+    function testSetEjector_Valid() public {
+        address newEjector = address(uint160(uint256(keccak256("newEjector"))));
+        cheats.prank(serviceManagerOwner);
+        cheats.expectEmit(true, true, true, true, address(registryCoordinator));
+        emit EjectorUpdated(ejector, newEjector);
+        registryCoordinator.setEjector(newEjector);
+    }
```

Comment:
you should check that the ejector gets updated correctly after this call

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/142/commits/7eef90501cca5c18d0bd82e76d092013992e4f31